lib/srv/sess.go

```diff
@@ -1341,10 +1239,44 @@ func (s *session) SetLingerTTL(ttl time.Duration) {
 	s.lingerTTL = ttl
 }
 
+func (s *session) isStopped() bool {
+	select {
+	case <-s.stopC:
+		return true
+	default:
+		return false
+	}
+}
+
+// If after a short linger duration there are no active
+// parties in the session, end the session.
+func (s *session) lingerAndDie(party *party) {
+	s.log.Debugf("Session %v has no active party members.", s.id)
+
+	time.Sleep(s.GetLingerTTL())
```

Comment:
Any reason not to use `s.scx.srv.GetClock()` instead of `time` here like it was before?

Reply:
Thanks for catching this, fixed